controllers/questionController.js

```diff
@@ -0,0 +1,61 @@
+import {
+   save,
+   getAllQuestions,
+   getQuestionByID,
+   getQuestionsFromIdArray,
+   updateQuestionByID,
+   deleteQuestionByID
+} from '../models/questionModel';
+
+import AppError from '../errors/AppError';
+
+
+
+exports.addNewQuestion = async(req, res, next) => {
```

Comment:
Please create const here and export all modules at the bottom of the file

Reply:
added const and exports at the bottom